examples/recognition/basic_async.py

```diff
@@ -78,7 +78,13 @@ async def expect(*event_classes):
                 )
             )
 
-            await send(client.set_params(speech_language="fr", no_input_timeout=5000))
+            await send(
+                client.set_params(
+                    speech_language="fr",
+                    no_input_timeout=5000,
+                    logging_tag="basic_async_test",
```

Comment:
What is this `logging_tag`?

Reply:
https://app.shortcut.com/allomedia/story/46544/gather-the-logging-tag-header-and-add-it-to-the-metric-payload

Reply:
The difference between `session-id` and `logging-tag` is that the former is for the complete session and the latter is used to tag a contiguous sequence of interactions within the session.

Reply:
Thanks for the explanations